typedb-src/modules/ROOT/pages/overview.adoc

```diff
@@ -2,25 +2,25 @@
 :keywords: typedb, typeql, documentation, overview, introduction
 :longTailKeywords: documentation overview, learn typedb, learn typeql, typedb schema, typedb data model
 :pageTitle: Documentation overview
-:summary: A birds-eye view of TypeQL and TypeDB
+:summary: A bird's-eye view of TypeQL and TypeDB
 
-This is the TypeDB server documentation section. It has the following sections:
+TypeDB server documentationm, which contains the following subsections:
```

Comment:
```suggestion
TypeDB server documentation, which contains the following subsections:
```
Yes, we will need to discuss the terminology of the docs and how we divide content by sections/subsections, etc.